packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -30,6 +30,14 @@ class CreateCommand extends Command {
       defaultsTo: true,
       help: 'Whether to run "pub get" after the project has been created.'
     );
+    argParser.addFlag(
+      'with-driver-test',
```

Comment:
I think it will be a mix for something serious, but for people simply trying out the technology it will be none.

There are two reasons I decided to make this opt-in:

1. To keep simple things simple. One great thing about Dart is that you don't need a project generator to get started. As soon as you have a project generator feature creep tends to get the setup to a point that you can no longer live without a generator. Today, you can `flutter create -o myapp` and literally read through all the generated files in less than 5 minutes and understand what's going on. Good luck doing that with native Android/iOS SDKs.
1. Having tests immediately requires more maintenance. When someone just trying things out, they are likely to break the test as soon as they make the first change in the app and potentially trigger analyzer warnings in Atom.

This option is useful for people whose purpose is to learn how testing works in Flutter, or for those who are committed to building a production app.